README.md

```diff
@@ -27,7 +21,7 @@ $ make docs
 $ make help
 ```
 
-### 4. Definitions
```

Comment:
How irritated are you that you need to change the numbers every time? 🤭

Reply:
It's just this one time since I moved the `Test / Debug` part to the end.